packages/authenticator/amplify_authenticator/CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+## 1.5.0
+
+### Fixes
+- fix(authenticator): Beta channel fixes ([#3835](https://github.com/aws-amplify/amplify-flutter/pull/3835))
+- fix(authenticator): fix validate before trim bug ([#3809](https://github.com/aws-amplify/amplify-flutter/pull/3809))
+- fix(authenticator): use getCurrentUser to check sign in state
+
+### Features
+- feat: rename sendUserAttributeVerificationCode ([#3759](https://github.com/aws-amplify/amplify-flutter/pull/3759))
```

Comment:
*todo:* Let's remove this. This was my mistake. I should have broken this PR up into smaller commits and made the commits to the authenticator package a `chore` so that it would get ignored.